README.md

```diff
@@ -12,7 +12,7 @@ This project contains source code and supporting files for a serverless applicat
 This application creates a serverless Amazon Rekognition Custom Label Detection workflow which runs on a pre-defined schedule (note that the schedule is enabled by default at deployment). It demonstrates the power of Step Functions to orchestrate Lambda functions and other AWS resources to form complex and robust workflows, coupled with event-driven development using Amazon EventBridge.
 
 Solution Architecture Diagram:
-<img width="814" alt="image" src="https://user-images.githubusercontent.com/34427009/102906604-c5c34e80-446c-11eb-918e-42dabbcacc93.png">
+<img width="814" alt="Architecture Diagram" src="docs/Solution Architecture - Serverless Computer Vision Label Detection.png">
```

Comment:
```suggestion
<img width="814" alt="Architecture Diagram" src="docs/Solution%20Architecture%20-%20Serverless%20Computer%20Vision%20Label%20Detection.png">
```
Spaces don't seem to work in my preview - I think this should fix.

Also, would you be able to add a white background to the image? Since github added dark mode, a transparent background may make the image not visible for those users.